sql/core/src/main/scala/org/apache/spark/sql/execution/streaming/IncrementalExecution.scala

```diff
@@ -425,6 +425,29 @@ class IncrementalExecution(
               eventTimeWatermarkForEviction = iwEviction)
           ))
 
+      case UpdateEventTimeColumnExec(eventTime, delay, None,
```

Comment:
Could you elaborate more on why we need 2 entries for TransformWithStateInPandasExec? Also, can we have TransformWithStateInPandasExec specific comments? Thanks!

Reply:
The main difference is where we wrap the logical node. For Scala, we wrap the event time column and output a dataset type [here](https://github.com/apache/spark/pull/45376/files#diff-3437bb4bcaf2e18c305978985e474daab11e397dc5f4666c13c8e11da0d7180bR706). For Python, we wrap the event time column and returns a Dataframe type [here](https://github.com/apache/spark/pull/48124/files#diff-32b043dfe6b906fb2b240e8557f98b03a648ba792ce58d11b631744b34bcea71R481). So that results in a slightly difference in the physical plan when the query enters the `IncrementalExecution` case match here. I'll update this in the comment as well.

Reply:
We don't. Deleted a case class and will add more comments.